GOVERNANCE.md

```diff
@@ -0,0 +1,251 @@
+# Governance thin-edge.io
+
+This documents servers as a guiding document to ease technical decision making in the project as a whole.
+
+Some of its goals are:
+
+- Making sure that the different stakeholders are represented with their requirements
+- Making sure that contributors know how decisions are made and who they can address concerns/questions to
+- Making sure that team members know what is required of them for the project
+
+
+## The project vision
+
+In the [vision.md](./vision.md) the goal of the project is encoded.
+It specifies the different functional requirements and non-functional requirements of the project as a whole.
+All the stakeholders must agree on the vision upon joining.
+
+- The current stakeholders are: Software AG, and IFM
+
+Changes to the vision need to be done in a plenum of stakeholders, this makes sure that everyone is aware and agrees to the evolution of the project.
```

Comment:
I am more and more in favor of removing this. Thin-edge should be able to stand on its own, and not be controlled by any amount of 'stakeholders'.

However, I agree its a bit contentious, so I would be fine with leaving it in, for now.

Reply:
This makes sense but needs discussions with all stakeholders.

Reply:
After discussion, the agreement is that thin-edge is controlled by stakeholders (starting with SAG & IFM) and each stakeholder nominate a maintainer acting on their behalf. Changes to the vision will be done in a plenum of the stakeholders.

Reply:
I would consider resolved then!